tests/sepp/ValueGenerators/RandomStringGenerator.h

```diff
@@ -0,0 +1,44 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2022 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Manuel Pöter
+////////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include <cstdint>
+#include "ValueGenerator.h"
+
+#include "Basics/xoroshiro128plus.h"
+
+namespace arangodb::sepp::generators {
+
+struct RandomStringGenerator : ValueGenerator {
+  RandomStringGenerator(std::uint32_t size);
+
+  /// @brief Generates a value and writes it to the given Builder.
+  void apply(velocypack::Builder&) override;
```

Comment:
`Builder` is undeclared in this header, and only transitively made available.
Let's follow IWYU (include-what-you-use) and either include `velocypack/Builder.h` or forward-declare it.

Reply:
The forward declaration is in `ValueGenerator.h` which we rely on since we are deriving from `ValueGenerator` and this is an override of a pure-virtual function from `ValueGenerator`. So IMHO we do not need any additional includes or forward declarations in this file.